rtgui/batchqueuepanel.h

```diff
@@ -53,29 +55,32 @@ class BatchQueuePanel : public Gtk::VBox,
 
     IdleRegister idle_register;
 
+    std::atomic<bool> qStartStopState;
```

Comment:
Excellent. Please move this before `idle_register`, which should always be the last member in case someone forgets to call `IdleRegister::destroy()` in the d'tor.